org.opentosca.bus/org.opentosca.bus.application.api.jsonhttp/src/main/java/org/opentosca/bus/application/api/jsonhttp/route/Route.java

```diff
@@ -46,25 +46,25 @@ public void configure() throws Exception {
         // handle exceptions
 
         onException(Exception.class).handled(true).setBody(exchangeProperty(Exchange.EXCEPTION_CAUGHT))
-                                    .process(exceptionProcessor);
+            .process(exceptionProcessor);
 
         // invoke route
         from("rest:" + Route.BASE_ENDPOINT + Route.INVOKE_ENDPOINT
             + "?method=post").process(invocationRequestProcessor).to(Route.TO_APP_BUS_ENDPOINT).choice()
-                             .when(exchangeProperty(Exchange.EXCEPTION_CAUGHT).isNull())
-                             .process(invocationResponseProcessor).removeHeaders("*").otherwise()
-                             .process(exceptionProcessor);
+            .when(exchangeProperty(Exchange.EXCEPTION_CAUGHT).isNull())
+            .process(invocationResponseProcessor).removeHeaders("*").otherwise()
+            .process(exceptionProcessor);
 
         // isFinished route
         from("rest:" + Route.BASE_ENDPOINT + Route.POLL_ENDPOINT + "?method=get").process(isFinishedRequestProcessor)
-                                                                                 .to(Route.TO_APP_BUS_ENDPOINT)
-                                                                                 .process(isFinishedResponseProcessor)
-                                                                                 .removeHeaders("*");
+            .to(Route.TO_APP_BUS_ENDPOINT)
+            .process(isFinishedResponseProcessor)
+            .removeHeaders("*");
 
         // getResult route
         from("rest:" + Route.BASE_ENDPOINT + Route.GET_RESULT_ENDPOINT
             + "?method=get").process(getResultRequestProcessor).to(Route.TO_APP_BUS_ENDPOINT)
-                            .process(getResultResponseProcessor).removeHeaders("*");
+            .process(getResultResponseProcessor).removeHeaders("*");
 

```

Comment:
Why the reformat? Was the code style not followed as specified [here](https://github.com/OpenTOSCA/container/tree/master/docs/codestyle)?

Reply:
The codestlye is the one defined within the intellij project files on master, maybe a PR which establishes codestyle in all IDE project files would help here

Reply:
Maybe the provided codestyle settings in the docs folder should be updated/deleted then.